apps/emqx_bridge/src/emqx_bridge.erl

```diff
@@ -237,9 +237,15 @@ send_to_matched_egress_bridges_loop(Topic, Msg, [Id | Ids]) ->
     send_to_matched_egress_bridges_loop(Topic, Msg, Ids).
 
 send_message(BridgeId, Message) ->
-    {BridgeType, BridgeName} = emqx_bridge_resource:parse_bridge_id(BridgeId),
-    ResId = emqx_bridge_resource:resource_id(BridgeType, BridgeName),
-    send_message(BridgeType, BridgeName, ResId, Message, #{}).
+    {BridgeV1Type, BridgeName} = emqx_bridge_resource:parse_bridge_id(BridgeId),
```

Comment:
I think this won't work with the new action id (`action:kafka_producer:action_name:connector:...`):

https://github.com/emqx/emqx/blob/2d4b9a7b9d259c220ef81331f5ebae7a36659d20/apps/emqx_bridge/src/emqx_bridge_resource.erl#L109-L113

Maybe we could extend it like this:

https://github.com/thalesmg/emqx/blob/2b2f25f2fb6b0e257595f33fe02dc803c0911a08/apps/emqx_bridge/src/emqx_bridge_resource.erl#L109-L117

?

Reply:
Clarified internally, this change/fix will ship later.